components/raftstore-v2/src/operation/query/local.rs

```diff
@@ -184,17 +184,23 @@ where
             Ok(ReadRequestPolicy::StaleRead) => {
                 ReadResult::Ok((delegate, ReadRequestPolicy::StaleRead))
             }
-            // It can not handle other policies.
             // TODO: we should only abort when lease expires. For other cases we should retry
             // infinitely.
-            Ok(ReadRequestPolicy::ReadIndex) => ReadResult::Redirect,
+            Ok(ReadRequestPolicy::ReadIndex) => {
```

Comment:
Can the replica read be covered by Redirect as well?
Because for follower read,  it's same as the expired local lease.